skills/okf/scripts/validate.py

```
@@ -0,0 +1,130 @@
+def validate_reserved(path: Path, bundle_root: Path) -> list[Finding]:
+    findings: list[Finding] = []
+    relative = path.relative_to(bundle_root).as_posix()
+    text = path.read_text(encoding="utf-8")
+    if path.name == "log.md" and has_frontmatter(text):
+        findings.append(Finding(ERROR, "E_RESERVED_FRONTMATTER", relative, "log.md must not have frontmatter"))
+    if path.name == "index.md" and has_frontmatter(text):
+        frontmatter, _body = split_frontmatter(text)
+        if path.parent != bundle_root or set(frontmatter) != {"okf_version"}:
+            findings.append(Finding(ERROR, "E_RESERVED_FRONTMATTER", relative, "index.md frontmatter is only allowed at bundle root for okf_version"))
+    if path.name == "index.md" and BEGIN in text and END in text:
+        before, rest = text.split(BEGIN, 1)
```

Ruff (0.15.20)

[warning] Unpacked variable `before` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)